integration/cluster.go

```diff
@@ -420,7 +420,7 @@ func isMembersEqual(membs []client.Member, wmembs []client.Member) bool {
 
 func newLocalListener(t *testing.T) net.Listener {
 	c := atomic.AddInt64(&localListenCount, 1)
-	addr := fmt.Sprintf("127.0.0.1:%d.%d.sock", c+basePort, os.Getpid())
+	addr := fmt.Sprintf("127.0.0.1:%d%06d", c+basePort, os.Getpid())
```

Comment:
`%05d%05d` so that nested bridges won't collide and comment about 1.8 requiring numbers in ports?